install/install_openadapt.ps1

```diff
@@ -260,6 +260,8 @@ function GetGitCMD {
 
 ################################   SCRIPT    ################################
 
+Write-Host "Install Script Started..." -ForegroundColor Yellow
```

Comment:
I could not reproduce the bug #3, hence what we can do for now is to do add this line and next time if the script fails we can at least know that it had ran the first line (Write-Host simlar to echo) or not. 

We can put an issue for the same ii the script is exiting without any messages to ensure that this is still being an issue. :(